From bc160c45377ae513c4ed30722a40a086613a9fe0 Mon Sep 17 00:00:00 2001
From: David Zeuthen <davidz@redhat.com>
Date: Thu, 17 Mar 2011 10:34:25 -0400
Subject: [PATCH] Nuke supervised accounts

The org.freedesktop.Accounts mechanism no longer supports supervised
accounts as per the patch in

 https://bugs.freedesktop.org/show_bug.cgi?id=35390

Therefore, nuke supervised accounts from the user interface as well.

Signed-off-by: David Zeuthen <davidz@redhat.com>
---
 panels/user-accounts/data/account-dialog.ui       |    4 ----
 panels/user-accounts/data/user-accounts-dialog.ui |    4 ----
 panels/user-accounts/um-account-type.c            |    2 --
 panels/user-accounts/um-account-type.h            |    3 +--
 4 files changed, 1 insertions(+), 12 deletions(-)

diff --git a/panels/user-accounts/data/account-dialog.ui b/panels/user-accounts/data/account-dialog.ui
index fc75484..3ae2fc3 100644
--- a/panels/user-accounts/data/account-dialog.ui
+++ b/panels/user-accounts/data/account-dialog.ui
@@ -24,10 +24,6 @@
         <col id="0" translatable="yes" context="Account type">Administrator</col>
         <col id="1">1</col>
       </row>
-      <row>
-        <col id="0" translatable="yes" context="Account type">Supervised</col>
-        <col id="1">2</col>
-      </row>
     </data>
   </object>
   <object class="GtkDialog" id="dialog">
diff --git a/panels/user-accounts/data/user-accounts-dialog.ui b/panels/user-accounts/data/user-accounts-dialog.ui
index 71d0db3..43ecdbd 100644
--- a/panels/user-accounts/data/user-accounts-dialog.ui
+++ b/panels/user-accounts/data/user-accounts-dialog.ui
@@ -24,10 +24,6 @@
         <col id="0" translatable="yes" context="Account type">Administrator</col>
         <col id="1">1</col>
       </row>
-      <row>
-        <col id="0" translatable="yes" context="Account type">Supervised</col>
-        <col id="1">2</col>
-      </row>
     </data>
   </object>
   <object class="GtkListStore" id="language-model">
diff --git a/panels/user-accounts/um-account-type.c b/panels/user-accounts/um-account-type.c
index 55064a8..30aeec7 100644
--- a/panels/user-accounts/um-account-type.c
+++ b/panels/user-accounts/um-account-type.c
@@ -35,8 +35,6 @@ um_account_type_get_name (UmAccountType account_type)
                 return C_("Account type", "Standard");
         case UM_ACCOUNT_TYPE_ADMINISTRATOR:
                 return C_("Account type", "Administrator");
-        case UM_ACCOUNT_TYPE_SUPERVISED:
-                return C_("Account type", "Supervised");
         default:
                 g_assert_not_reached ();
         }
diff --git a/panels/user-accounts/um-account-type.h b/panels/user-accounts/um-account-type.h
index 1e42e00..8967f2b 100644
--- a/panels/user-accounts/um-account-type.h
+++ b/panels/user-accounts/um-account-type.h
@@ -26,8 +26,7 @@ G_BEGIN_DECLS
 
 typedef enum {
   UM_ACCOUNT_TYPE_STANDARD,
-  UM_ACCOUNT_TYPE_ADMINISTRATOR,
-  UM_ACCOUNT_TYPE_SUPERVISED
+  UM_ACCOUNT_TYPE_ADMINISTRATOR
 } UmAccountType;
 
 const gchar *um_account_type_get_name (UmAccountType account_type);
-- 
1.7.4.1


